datashader/spatial/viewshed.py

```diff
@@ -31,13 +31,14 @@
 
 # this value is returned by findMaxValueWithinDist() if there is no key within
 # that distance
-SMALLEST_GRAD = -9999999999999999999999.0
+SMALLEST_GRAD = float('-inf')
 
 PROJ_LL = 0
 PROJ_NONE = -1
 
 PI = math.pi
```

Comment:
Why not just `from math import pi` and then use `pi`; why `PI`?

Reply:
@jbednar Yes...all hold overs from C++ code.  I marked them for refactor and didn't hit it yet, but let's hold off on merging this and I'll refactor today.

Reply:
@jbednar on second look, I like having the PI alias because its used everywhere, but it doesn't need to be so explicit...I've refactored accordingly and use an import alias instead of a variable

Reply:
Ok.